staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_patch.go

```diff
@@ -33,6 +33,12 @@ var (
 	// Example: root:org:ws/some-name
 	wildcardClusterNameRegex = regexp.MustCompile(`^([^/]+)\/.+`)
 
+	// matches shard/cluster/remainder, capturing shard and cluster.
+	//
+	// Example: amber/root:org:ws/some-namespace/some-name
+	// Example: amber/root:org:ws/some-name
+	wildcardShardClusterNameRegex = regexp.MustCompile(`^([^\/]+)\/([^\/]+)\/.+`)
```

Comment:
this does not match `amber/root:org:ws/some-name`, does it?

Reply:
I think it does, let me check.

Reply:
looks okay - https://go.dev/play/p/OFjiSce2jCC